charts/opentelemetry-collector/Chart.yaml

```diff
@@ -11,4 +11,4 @@ maintainers:
   - name: dmitryax
   - name: TylerHelmuth
 icon: https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png
-appVersion: 0.59.0
+appVersion: 0.59.0
```

Comment:
nl

Reply:
fixed, updated the chart version, and added back the newline at eof